syntax  ="proto3";

package Message ;

service VarifyService{
    rpc GetVarifyCode(GetVarifyreq) returns (GetVarifyRsp){}
}
message GetVarifyreq
{
    string email=1;

}
message GetVarifyRsp
{
    int32  error=1;
    string email=2;
    string code =3;
}